### DISCUSSION TITLE
[Help] is relative line numbers available?

### DISCUSSION THREAD

**Discussion by CyprienRicque** [mickael-menu/ShadowVim#60] [Help] [answered]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

Hello,

Actually I don't think it is possible from what I understood, but just to make sure: 
Is there any way to set relativenumber?

Thanks for your work!!

Cyprien

**Comment by mickael-menu** [comment, marked as answer]
Partly, it won't show in Xcode because ShadowVim doesn't control the UI, but relative numbers should work with count commands.

**Comment by zenangst** [comment]
Good news, Apple added relative line numbers to Xcode 15.3 🤘
